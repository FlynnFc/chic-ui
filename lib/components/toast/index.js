import { toast } from './toastmanager';
export * from './toastmanager';
export default toast;
//# sourceMappingURL=index.js.map