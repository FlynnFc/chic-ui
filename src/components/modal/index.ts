import Avatar from './modal';

export * from './modal';

export default Avatar;
