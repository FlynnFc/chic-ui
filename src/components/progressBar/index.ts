import ProgressBar from './progressBar';
export * from './progressBar';
export default ProgressBar;