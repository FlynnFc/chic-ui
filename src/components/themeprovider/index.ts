import ThemeProvider from './themeprovider';

export * from './themeprovider';

export default ThemeProvider;