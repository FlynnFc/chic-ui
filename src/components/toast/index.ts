import { toast } from './toastmanager';

export * from './toastmanager';

export default toast;