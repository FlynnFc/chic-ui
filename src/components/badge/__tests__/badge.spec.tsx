import Badge from '..';
import { Avatar } from 'components';
import { render } from 'test/utils';

describe('Badge Component', () => {
  it('Renders Correctly', () => {
    const BadgeGroup = render(
      <>
        <div>
          <Badge type="danger" offset={[0, 0]} count={3} showCount>
            <Avatar src="chic_copy.png" />
          </Badge>
          <Badge type="success" offset={[-13, 11]} count={0} showZero>
            <Avatar src="chic_copy.png" />
          </Badge>
        </div>
      </>
    );
    expect(BadgeGroup).toMatchSnapshot();
  });
});
