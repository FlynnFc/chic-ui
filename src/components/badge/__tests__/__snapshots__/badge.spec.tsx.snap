// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge Component Renders Correctly 1`] = `
<div>
  <div
    className="sc-bdfBQB ntxAg"
  >
    <span
      className="sc-dlfnuX kcUYsq"
      src="chic_copy.png"
    >
      <img
        className="sc-hKgJUU fkNrNr"
        src="chic_copy.png"
      />
    </span>
    <div
      className="sc-gsTEea blCbYG"
      offset={
        Array [
          0,
          0,
        ]
      }
      type="danger"
    >
      3
    </div>
  </div>
  <div
    className="sc-bdfBQB ntxAg"
  >
    <span
      className="sc-dlfnuX kcUYsq"
      src="chic_copy.png"
    >
      <img
        className="sc-hKgJUU fkNrNr"
        src="chic_copy.png"
      />
    </span>
    <div
      className="sc-gsTEea kCJSgB"
      offset={
        Array [
          -13,
          11,
        ]
      }
      type="success"
    />
  </div>
</div>
`;
